packages/bot-web-ui/src/stores/flyout-help-store.js

```diff
@@ -50,6 +50,30 @@ export default class FlyoutHelpStore {
         }
     };
 
+    getHelpContent = async block_node => {
+        let block_content;
+
+        if (block_node) {
+            const target_blocks = this.xml_list_group[block_node];
+            const block_type = target_blocks[0].getAttribute('type');
+            const help_string_obj = await import(/* webpackChunkName: `[request]` */ '@deriv/bot-skeleton');
+            block_content = help_string_obj.default[block_type];
+        }
+
+        return block_content;
+    };
+
+    getFilledHelpContents = async blocks_Type => {
+        const blocks_content = await Promise.all(blocks_Type.map(block => this.getHelpContent(block)));
+        return blocks_content.map((key, index) => (key != null ? index : null)).filter(value => value != null);
+    };
+
+    getNextHelpContentIndex = async direction => {
```

Comment:
`direction` is a boolean and should be named as such. For booleans we use `is_`, `has_`, or `should_` prefixes, e.g. `is_next` or `is_previous` would be more appropriate here.

Reply:
done